[POKT] Shannon Beta & MainNet Endpoints

Olshansk opened the pull request:
- Replace `poktroll` with `pocket`
- Split `poktroll` into `pocket-mainnet` and `pocket-testnet`
- Update the icon

editaahn commented on cosmos/pocket-mainnet.json:
the file name should be `pocket`

editaahn commented on images/poktroll/chain.jpg:
could you please use a png file instead?

editaahn reviewed (commented):
Please add images to directories matching each json.

Olshansk commented:
@editaahn what I did:

1. Added images in the right directories
2. Using png instead of jpg
3. Added `pocket-testnet` which points to `pocket-beta` (we have alpha testnets I'm not including here)
4. Kept `pocket-mainnet` to point to `pocket

Not 100% sure if (3) & (4) are okay given best practices in this repo, so please let me know!

okdas commented on cosmos/pocket-testnet.json:
```diff
@@ -0,0 +1,49 @@
+
+  "chainId": "pocket-beta",
+  "chainName": "Pocket Network",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/pocket/chain.jpg",
```
```suggestion
  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/pocket-testnet/chain.png",
```
jpg -> png

Also, the directory where the `chain.png` file is located should be renamed from `pocket-mainnet` to `pocket`.

okdas commented on cosmos/pocket-testnet.json:
```diff
@@ -0,0 +1,49 @@
+{
+
+  "chainId": "pocket-beta",
+  "chainName": "Pocket Network",
```
```suggestion
  "chainName": "Pocket Network Testnet",
```

okdas commented on cosmos/pocket.json:
```diff
@@ -1,9 +1,10 @@
-  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/poktroll/chain.png",
-  "rpc": "https://shannon-testnet-grove-seed-rpc.poktroll.com",
-  "rest": "https://shannon-testnet-grove-seed-api.poktroll.com",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/pocket/chain.jpg",
```
```suggestion
  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/pocket/chain.png",
```

jpg -> png

Also, the directory where the `chain.png` file is located should be renamed from `pocket-mainnet` to `pocket`.
<img width="623" alt="Screenshot 2025-04-02 at 3 24 28 PM" src="https://github.com/user-attachments/assets/fd6c06ef-763a-4c00-ab3a-d7b6b8c33b6c" />

This is the file we're referencing here.

okdas commented:
@Olshansk please check if my suggestions make sense.
A couple of questions:
1. Thoughts on adding `"coinGeckoId": "pocket-network",`? Goes to each `currencies` (`feeCurrencies`, `stakeCurrency`).
2. Similarly, I think we should add `"coinImageUrl": "... same image as chainSymbolImageUrl ..."` to each `currencies. From the README.md:
> If `coinImageUrl` is not provided in `currencies`, Keplr doesn't show the token image on the wallet.
WDYT?

I can implement both - just let me know.

okdas commented on cosmos/pocket-testnet.json:
```diff
@@ -0,0 +1,49 @@
+{
+
+  "chainId": "pocket-beta",
```
The name of the file and the images/ directory should match chain id. https://github.com/chainapsis/keplr-chain-registry/actions/runs/14231617866/job/39883259588?pr=1081

editaahn reviewed (commented):
Hey, @Olshansk @okdas Here are additional requests!
- Please resize the images to 256*256px.
- Remove unused svg files.
- The wallet shows the token images if you provide `coinImageUrl` to `currencies`. it will be good for the mainnet chain to have them as well.
- if you want to provide coinGeckoId, please add it to mainnet only. Adding same coinGeckoId to testnet token could cause confusion.

Olshansk commented:
Thanks @editaahn! Just tackled everything you suggested.

Olshansk commented:
@editaahn I would love to get this over the finish line, especially as we are getting close to merging in [1] as well.

Let me know if this requires any further changes!

[1] https://github.com/cosmos/chain-registry/pull/6083

editaahn commented:
Thank you for the contribution!
